src/applications/letters/routes.jsx

```diff
@@ -20,7 +21,11 @@ const letterPageFeatureFlagWrapper = (
 const routes = (
   <Routes>
     <Route path="/" element={<App />}>
-      <Route index element={letterPageFeatureFlagWrapper} />
+      <Route element={<LetterPageWrapper />}>
+        <Route element={<Main />}>
+          <Route index element={letterPageFeatureFlagWrapper} />
+        </Route>
+      </Route>
```

Comment:
Why have the wrapper outside of the feature flag check?  This makes me nervous that some of the extra text might be exposed outside of the feature flag's domain.  I would like to if possible keep everything new going in for the sake of the redesign work within the control of the FF so nothing has the potential to be exposed without it.